fdbserver/include/fdbserver/workloads/MetaclusterConsistency.actor.h

```diff
@@ -220,6 +220,19 @@ class MetaclusterConsistencyCheck {
 				// allocation)
 				++clusterAllocated[entry.assignedCluster.get()];
 			}
+
+			// If the rename pair is present, it should be in the map and match our current entry
+			if (entry.renamePair.present()) {
+				auto pairMapEntry = managementMetadata.tenantMap[entry.renamePair.get()];
+				ASSERT(pairMapEntry.id == entry.id);
+				ASSERT(pairMapEntry.prefix == entry.prefix);
+				ASSERT(pairMapEntry.encrypted == entry.encrypted);
+				ASSERT(pairMapEntry.configurationSequenceNum == entry.configurationSequenceNum);
+				ASSERT(pairMapEntry.assignedCluster.present());
+				ASSERT(pairMapEntry.assignedCluster.get() == entry.assignedCluster.get());
+				ASSERT(pairMapEntry.renamePair.present());
+				ASSERT(pairMapEntry.renamePair.get() == name);
+			}
```

Comment:
We only need to check these properties in one place (I think the tenant consistency check makes sense). The thing we need to handle here is the fact that the tenant map on the management cluster might include extra tenants from the rename that aren't present on the data cluster.

Reply:
I think we still need to handle the case in metacluster consistency where we have two tenants present on the management cluster for only one on the data cluster. Otherwise, if we run the consistency check with a partially completed rename, it will fail thinking there is an extra tenant.

Instead if we see a tenant with a rename pair on the management cluster, we should not fail if the tenant is missing on the data cluster. We could also verify that the pair tenant is present on the data cluster in that case (if we are in the renaming_from or renaming_to states; the removing state is allowed to have neither tenant).